.github/workflows/.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
do we need files in this folder and the other .suo file?

Reply:
There are two PR to do:
* one PR in main branch to check in the documentation source, only folder `onnx/docs` (without `onnx/docs/build` and `onnx/docs/onnx_doc_folder`) should be checked in.
* another PR in branch gh-pages with folder `html` (the generated documentation)

Reply:
add "Doc" button in the ONNX.ai main manual to access to the generate content?